server/src/Dao/RevisionDao.php

```diff
@@ -50,9 +51,13 @@ public function getUserRevisionsInCommonPages(
 	 * Get common edited pages between multiple users
 	 *
 	 * @param int[] $users
+	 * @param \DateTime|null $startDate
+	 * @param \DateTime|null $endDate
 	 * @return array
 	 */
-	public function getUsersCommonPages( array $users ) {
+	public function getUsersCommonPages(
+		array $users, \DateTime $startDate = null, \DateTime $endDate = null
```

Comment:
nit-pick typehint the `\DateTimeInterace`
https://secure.php.net/manual/en/class.datetimeinterface.php